### DISCUSSION TITLE
Multiple config files in a single repo

### DISCUSSION THREAD

**Discussion by jdechicchis** [renovatebot/renovate#9216] [Request Help]
I'm not sure if this is a supported use case, but I was wondering if there could be multiple renovate config files in a single repo. My use case is as follows:

- Monorepo with team-based ownership of subfolders
- Ideally, renovate could be executed for each team with their own configuration such as managers, exclude paths, package ignores etc as well as assignees

Maybe I'm going about this the wrong way, but the straightforward solution seemed to be to have multiple renovate configs each with a single `includePaths`. This works except that renovate closes pull requests which it thinks should be cleaned up. For example, if config A runs an npm manager and config B runs a gomod manager, then running config B after config A seems to close all npm related pull requests.

Thank you in advance for your help!

**Comment by rarkins** [comment]
You cannot have multiple config files, but you can control your configuration fully through `packageRules`, each of which can have a `matchPaths` or `matchFiles` field to limit the rule. You can also add a "description" field to each rule to state who owns it and why it's there, to avoid confusion.

**Reply by jdechicchis** [reply]
Thank you for the quick reply! `packageRules` does seem to cover the use case, but is there a way to have assignees for different paths? I see there's `assigneesFromCodeOwners` but that seems to assume codeowners are set up

**Reply by rarkins** [reply]
`assignees` is configurable using `packageRules` too, so yes
